### DISCUSSION TITLE
disconnect tunnel

### DISCUSSION THREAD

**Discussion by rootrrv** [isontheline/pro.webssh.net#283] [General]
I bought the webssh on 08/20/2017 (iphone SE). Previously, multiple tunnels worked, but now writes "Please disconnect tunnel before launching another"

**Comment by isontheline** [comment]
Hello @rootrrv 👋

Thanks for your support and purchase!

Could you describe your use case? 
Why multiple tunnels at the same time?
Doesn't the dynamic port forwarding could fit your needs?

Have a nice day ☀️

**Comment by rootrrv** [comment]
I am redirecting tunnels through two servers because access is restricted in other options.
The first connection redirects remote tunnel 22 to local 8222. Then the second connection connects to local port 8222 and forwards the tunnel to RDP on the remote LAN.

**Reply by isontheline** [reply]
I'm sorry I don't understand your two port forwards (I'm tired this night).

Could you produce a little draw?

Thank you Roman

**Reply by isontheline** [reply]
Doesn't the RDP 3389 port can't be forwarded with only one tunnel like this?
3389:IP_OF_RDP_SERVER:3389

**Comment by rootrrv** [comment]
1: my phone -> my_public_server:22   => port forward:  8222:my_private_server:22
2: my phone -> localhost:8222             => port forward: 83389:my_pc:3389       (RDP)

3: my phone RDP client -> localhost:83389

**Reply by rootrrv** [reply]
This scheme has worked fine in the past. I used the keys and had quick access to my PC.

Thanks for the quick answers. I'm in no hurry.
Good night.

**Reply by isontheline** [reply]
Multiple tunnels have been removed in October 2020 but I think there is a bug because I only want to remove the multiple dynamic port forwarding and not the "simple" port forwarding.

I'm opening an issue right now : #284 

I will try to check if I can allow multiple forwards at the same time by using the new NEVPN component.

**Reply by rootrrv** [reply]
Many thanks! You have a great app! I will wait for a solution to the problem!

**Reply by isontheline** [reply]
Thank you a lot for your support @rootrrv Roman!
